src/main/webapp/style.css

```diff
@@ -0,0 +1,42 @@
+.navbar-background{
+    background-color: #d2fdff;
+}
+body {
+    background-color: hsl(19, 86%, 69%);
+}
+h1 {
+    padding: 10px;
+}
+.content {
+    align-content: center;
+    color: #303c6c;
+    text-align: center;
+}
+.button-background {
+    background-color: #303c6c;
+    color: white;
+}
+.nav-text-style {
+    color: #303c6c;
+    font-size: 30px;
```

Comment:
Instead of using `px` for the `font-size`, a scaling unit like `em` would be better because it scales with the size of the screen.